src/components/molecules/widgets/AccidentCountByDriverType.tsx

```diff
@@ -1,11 +1,14 @@
 import React, { FC } from 'react';
 import { IWidgetAccidentCountByDriverType } from '../../../models/WidgetData';
-
+import PieChartView from '../PieChartView';
 interface IProps {
   data: IWidgetAccidentCountByDriverType;
 }
+const DRIVER_TYPE = 'driver_type';
```

Comment:
put `consts` after `imports` and before other things in this file (like interfaces)